common/changes/bfix-removeSearchBoxLineHeight_2017-01-06-00-24.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Remove line-height to show correct cursor size in iOS",
```

Comment:
Mind tweaking this to say the following? (that way it will be more understandable in the release notes.)

SearchBox: Removing line-height to show correct cursor size in IOS.